### DISCUSSION TITLE
Voting in discussions

### DISCUSSION THREAD

**Discussion by ysthakur** [Vyxal/Vyxal#208] [General]
For top-level answers, can we use upvotes instead of thumbs up emojis to vote? Of course, comments on those will need emojis.

Please upvote this if you want to make everyone use upvotes, use the thumbs-up emoji to make everyone use the thumbs-up emoji, and use another emoji if you don't want to enforce anything.

**Comment by lyxal** [comment]
Question: do you suggest people remove the automatic upvote on answers?

**Reply by ysthakur** [reply]
No, because if we want to know how many people support something, we won't have to look at the votes and then add 1 for the answerer, we can just look at the number of votes and be done.
